README.md

```diff
@@ -35,28 +35,27 @@
 ### Script 
 1. Download this repository extract and open the folder
 2. Run `newproject.sh` script to create a copy with the name you want 
-3. On `app`, change the package on the `mode` lazy var
+    * Example: `./newproject.sh -p com.example.project -t project` 
```

Comment:
Even if it's not a requirement, it's a good practice for the title to start with Uppercase.